source/guides/clear/stateless.rst

```diff
@@ -65,14 +65,13 @@ Default configurations
 Software in |CL| provides default configuration values so that it is
 immediately functional, whenever it is appropriate to do so.
 
-|CL| distributed software packages may be directly modified to include default
+|CL| distributed software may be directly modified to include default
```

Comment:
Consider rephrasing this to emphasize that |CL| modifies the software. Right now it could be interpreted as the user may modify...

Reply:
Good point. I'll wait on @bktan8.

Reply:
How's this?
```
Software in |CL| provides default configuration values so that it is
immediately functional, whenever it is appropriate to do so.

If an upstream software puts default configurations in multiple locations such as :file:`/usr/` and :file:`/etc`, it will be modified by the |CL| distro to comply with the stateless design.  Also, some default configurations may be modified to close security loopholes.  The defaults will reside under :file:`/usr/share/defaults`.  These files can be referenced as templates
for customization.
```

Reply:
Nicely worded.

You could simplify by removing this sentence:
> Also, some default configurations may be modified to close security loopholes.

with this in the first sentence:

> Software in |CL| provides *secure* default configuration values so that it is

Reply:
Agreed. Nice job, @bktan8. I prefer the first draft on security loopholes because it addresses the importance of integrating CVE patches from upstream. 

I don't understand the phrase, "whenever it is appropriate to do so."

In my reading, the sentence stands strong without that phrase. 
Was a shade of meaning lost by removing it?

Reply:
@puneetse - I'd like to keep the first sentence as is.  The sentence about security closing loopholes is needed in order to emphasize that we purposely do that.  Simply adding "secure" to the first sentence, I'm afraid, will get lost in the messaging.

@mvincerx - regarding "whenever it is appropriate to do so" - some software we distribute isn't immediately functional.   The user has to do additional configuration to get it running.  For example, nginx.  After installing the bundle, you don't want to start it automatically because you have to configure it for your specific web server.  So it's "whenever it's appropriate to do so."

Reply:
Oh, sure. In that case, I recommend using some of your response above. 

"...immediately functional,  except some bundles require additional configuration.

I'm open to your version, too. I just found the phrase "whenever it is appropriate to do so" disconnected from the first clause of the sentence. Wild goats eat green grass. You know. That kind of non-sequitur.

Reply:
ok - all suggested incorporated.

Reply:
LGTM. 

@puneetse?